pkg/aws/util_test.go

```diff
@@ -0,0 +1,86 @@
+package aws
+
+import (
+	api "github.com/gardener/machine-controller-manager-provider-aws/pkg/aws/apis"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Util", func() {
+
+	Describe("#NewSession", func() {
+		type setup struct {
```

Comment:
The `setup`, `action` and `expect` was custom pattern I thought made sense in the MCM controller unit tests because the test data was such deeply nested objects. But here the test data is simpler, you can directly pass the `secret` as the test data to the func instead of this `setup` business. WDYT?

Reply:
My bad. I got used to the setup, action and expect pattern. 😄 Fixed it. https://github.com/gardener/machine-controller-manager-provider-aws/pull/2/files#diff-fbcb9d3dc85f67d997c1adc27b00963dR14